includes/common-pages.php

```diff
@@ -205,8 +205,9 @@ public static function edit_page() {
 	 * @since 4.1.8 Added Exclude roles arguments.
 	 */
 	public static function activities_page() {
-		$roles = array_keys( ap_opt( 'activity_exclude_roles' ) );
-		$args  = array();
+		$roles_exclude = ap_opt( 'activity_exclude_roles' );
+		$roles         = ! is_array( $roles_exclude ) ? array_keys( $roles_exclude ) : array();
```

Comment:
@mebishalnapit There is a bug in this line, condition should be opposite.

Reply:
@rahularyan Fixed.